### PR TITLE
Added org.surge_synth_team.surge-xt

### PR DESCRIPTION
**Pull request by hfiguiere**
This is meant to replace https://github.com/flathub/org.freedesktop.LinuxAudio.Plugins.Surge-XT/
It does the following
- Build Surge-XT standalone
- Build the plugins like https://github.com/flathub/org.freedesktop.LinuxAudio.Plugins.Surge-XT/

For the user the benefit is the standalone version of Surge is now available. The plugins as still here and there is no action required.

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
  - Filesystem permission for the standalone are set to what works. JUCE doesn't deal with XDG well.
  - Pipewire is required for JACK
  - X11 only
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
  - The preious issue when the plugins were packaged https://github.com/surge-synthesizer/surge/issues/5429
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
  - The app-id is as requested: https://github.com/surge-synthesizer/surge/issues/5429#issuecomment-974054403
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*
  - I'll see about getting the desktop and appstrea files upstream for the standalone app.

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
bot, build org.surge_synth_team.surge-xt

**Comment by hfiguiere** [comment]
Like for the plugins only repo, I think @baconpaul should be added to the list committers.

**Comment by hfiguiere** [comment]
bot, build org.surge_synth_team.surge-xt

File: org.freedesktop.LinuxAudio.Plugins.Surge-XT.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="addon">
+  <id>org.freedesktop.LinuxAudio.Plugins.Surge-XT</id>
+  <extends>org.surge_synth_team.surge-xt</extends>
+  <name>Surge XT Synthesizer (LV2/VST3)</name>
+  <summary>Surge XT synthesizer and effects LV2/VST3</summary>
```
Comment (abique) [inline org.freedesktop.LinuxAudio.Plugins.Surge-XT.metainfo.xml:6]: Isn't surge also available as vst2?

**Comment by barthalion** [comment]
@hfiguiere Please let me know when you want to merge it.

**Comment by hfiguiere** [comment]
It's ready.

**Comment by hfiguiere** [comment]
bot, build org.surge_synth_team.surge-xt

File: org.freedesktop.LinuxAudio.Plugins.Surge-XT.metainfo.xml (hunk 1 above)
Comment (baconpaul) [reply org.freedesktop.LinuxAudio.Plugins.Surge-XT.metainfo.xml:6]: No.  But it is available as a clap

**Pull request closed without merging**